webpack.config.js

```diff
@@ -36,7 +36,7 @@ module.exports = (env = {NODE_ENV: process.env.NODE_ENV || 'production'}) => ({
     'eval-cheap-module-source-map' : 'source-map',
   devServer: {
     https: true,
-    port: 8000,
+    port: process.env.PORT || 8000,
```

Comment:
Using the .env for PORT inside the web pack config file.

Reply:
looks good

Reply:
@arun3528, Thanks..